_includes/footer.html

```diff
@@ -5,53 +5,82 @@
   </div>
   {% endunless %}
 
-  <div class="usa-footer__secondary-section bg-primary">
-    <div class="grid-container">
-      <div class="display-flex flex-align-center margin-bottom-4">
-        <a href="https://www.whitehouse.gov/omb/"
-          ><img
-            class="width-10 display-block"
-            src="{{ '/assets/img/eop-omb.png' | relative_url }}"
-            alt="Office of Management and Budget"
-        /></a>
-        <div class="separator display-flex flex-align-center">
-          <a href="https://digital.gov"
-            ><img
-              class="height-3 display-block"
-              src="{{ '/assets/img/digital-gov-logo--white.png' | relative_url }}"
-              alt="Digital.gov"
-          /></a>
+
+
+  <div class="usa-identifier">
+    <section class="usa-identifier__section usa-identifier__section--masthead" aria-label="Agency identifier">
+      <div class="usa-identifier__container">
+        <div class="usa-identifier__logos">
+          <a href="https://www.gsa.gov" class="usa-identifier__logo">
+            <img class="usa-identifier__logo-img" src="{{ '/assets/img/gsa-logo-w100.png' | relative_url }}" alt="GSA logo" role="img">
```

Comment:
Is there an SVG version of the logo available? Would look better on higih res screens.

Reply:
Still looking for a high-res version...